target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -181,6 +181,9 @@ ath79_setup_interfaces()
 	tplink,tl-wr941-v2)
 		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan"
 		;;
+	tplink,tl-wr710n-v1)
```

Comment:
The ubnt,routerstation and the glinet,ar150, glinet,ar300m and glinet,gl-x750 seem to share the same setting.
I wonder why nobody has tried to merge them together.

Reply:
Now it's merged with glinet etc